.dockerignore

```diff
@@ -0,0 +1,2 @@
+virtualenv_run
+.git
```

Comment:
per IRC, we should probably keep this.

I think `virtualenv_run` should probably me changed to `.tox` though?